test/utils/introspection/SupportsInterface.behavior.js

```diff
@@ -30,6 +30,16 @@ const INTERFACES = {
     'onERC1155Received(address,address,uint256,uint256,bytes)',
     'onERC1155BatchReceived(address,address,uint256[],uint256[],bytes)',
   ],
+  ERC1363: [
+    'transferAndCall(address,uint256)',
+    'transferAndCall(address,uint256,bytes)',
+    'transferFromAndCall(address,address,uint256)',
+    'transferFromAndCall(address,address,uint256,bytes)',
+    'approveAndCall(address,uint256)',
+    'approveAndCall(address,uint256,bytes)',
+  ],
+  ERC1363Receiver: ['onTransferReceived(address,address,uint256,bytes)'],
+  ERC1363Spender: ['onApprovalReceived(address,uint256,bytes)'],
```

Comment:
```suggestion
```
These interfaces should not appear here. This helper is only for ERC165 checks, which is not part of the receiver and spender interfaces

Reply:
Right. Removed.